src/main/java/seedu/address/logic/commands/alias/ListAliasCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands.alias;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.AliasMapping;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+
+public class ListAliasCommand extends Command {
+    public static final String COMMAND_WORD = "aliases";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": List all user-defined aliases.";
+
+    public static final String MESSAGE_EMPTY_ALIAS = "No aliases have been created yet.";
+
+    public static final String MESSAGE_SUCCESS_HEADER = "All %d aliases listed: \n";
+
+    @Override
+    public CommandResult execute(Model model) {
```

Comment:
Perhaps this method should have JavaDocs even though it's an overridden method?

Saw this in other files.